engine/flink/components/base/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/transformers.scala

```diff
@@ -73,7 +73,8 @@ object transformers {
                           windowLength: Duration,
                           variableName: String,
                           tumblingWindowTrigger: TumblingWindowTrigger,
-                          explicitUidInStatefulOperators: FlinkCustomNodeContext => Boolean
+                          explicitUidInStatefulOperators: FlinkCustomNodeContext => Boolean,
+                          windowOffset: Option[Duration] = None
```

Comment:
2 things here:
- why in line 66 we don't use windowOffset?
- maybe windowOffset shouldn't be Optional and it should be determined earlier - fromConfig.getOrElse(default).getOrElse(zero)? So we can have single source of truth?

Reply:
1. fixed
2. I think it should be optional, since it is optional in Flink (it gives to factory methods to create Window, one without this parameter)